charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml

```diff
@@ -425,6 +425,9 @@ spec:
 {{- if .Values.prometheus.prometheusSpec.sampleLimit }}
   sampleLimit: {{ .Values.prometheus.prometheusSpec.sampleLimit }}
 {{- end }}
+{{- if .Values.prometheus.prometheusSpec.enforcedKeepDroppedTargets }}
+  enforcedKeepDroppedTargets: {{ .Values.prometheus.prometheusSpec.enforcedKeepDroppedTargets }}
```

Comment:
Please add this to values with an description like this: https://github.com/prometheus-community/helm-charts/blob/1f70a6b763a245f135f34c5a71f7149e3926d84c/charts/kube-prometheus-stack/values.yaml#L3798-L3801

Reply:
Done. I have added the changes.